### ISSUE TITLE
Bei zu langem Schlagwort-Text liegt Titelzeile über Überschrift der Schlagwort-Seite

### ISSUE THREAD

**Issue by diesocke** [Piratenpartei/Pirate-Rogue#181]
Wenn man auf ein Schlagwort klickt, erscheint die Schlagwort-Seite. Hat man ein langes Schlagwort gewählt, z.B. "Bedingungsloses Grundeinkommen" ist die Überschrift nicht einzeilig, sondern bricht um, so dass die Überschrift hinter die Titelzeile rutscht.

**Comment by xwolfde** [comment]
Ich sehe da kein problem?
http://www.pirate-rogue.de/tag/dieses-schlagwort-ist-ueberlang-und-eigentlich-murks/

**Comment by diesocke** [comment]
Ja. Sehe ich dort jetzt auch nicht mehr. Allerdings scheint die Startseite betroffen zu sein, dass das Burgermenü verrutscht. Siehe http://www.sockenseite.de/wordpress/wp-content/uploads/2017/09/Screenshot_20170906-215256.png und http://www.sockenseite.de/wordpress/wp-content/uploads/2017/09/Screenshot_20170906-215301.png (Das Burgermenü taucht erst auf, wenn ich den Bildschirm etwas nach rechts verschiebe.)

**Comment by xwolfde** [comment]
Mist... muss ich nochmals schauen

**Comment by diesocke** [comment]
Tritt bei http://www.sockenseite.de/wordpress/wp-content/uploads/2017/09/Screenshot_20170908-071557.png bzw. http://www.sockenseite.de/wordpress/wp-content/uploads/2017/09/Screenshot_20170908-071601.png übrigens auch auf.
